samples/nrf9160/udp/src/main.c

```diff
@@ -0,0 +1,227 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <zephyr.h>
+#include <stdio.h>
+#include <modem/lte_lc.h>
+#include <net/socket.h>
+
+#define UDP_IP_HEADER_SIZE 28
+
+static int client_fd;
+static struct sockaddr_storage host_addr;
+static struct k_delayed_work server_transmission_work;
+
+K_SEM_DEFINE(lte_connected, 0, 1);
+
+static void server_transmission_work_fn(struct k_work *work)
+{
+	int err;
+	char buffer[CONFIG_UDP_DATA_UPLOAD_SIZE_BYTES] = {"\0"};
+
+	printk("Transmitting UDP/IP payload of %d bytes to the ",
+	       CONFIG_UDP_DATA_UPLOAD_SIZE_BYTES + UDP_IP_HEADER_SIZE);
+	printk("IP address %s, port number %d\n",
+	       CONFIG_UDP_SERVER_ADDRESS_STATIC,
+	       CONFIG_UDP_SERVER_PORT);
+
+	err = send(client_fd, buffer, sizeof(buffer), 0);
+	if (err < 0) {
+		printk("Failed to transmitt UDP packet, %d\n", err);
```

Comment:
s/transmitt/transmit

Plus, you should print `errno` value here, err will always be -1 in case of error.